tests/test_elem_validator_msg.py

```diff
@@ -52,8 +52,42 @@ def test_result_msg(self):
         def identity(arg):
             return arg
 
-        res_msg_fmt = "{} is {} on input {}"
+        res_msg_fmt = "{} is {} on input '{}'"
         for x in (True, False):
             expected_msg = res_msg_fmt.format(msg, str(x).lower(), x)
             # pylint: disable=no-member
```

Comment:
what is this pylint: disable:no-member stuff? necessary to have - or... ?

Reply:
It tells `pylint` to ignore its error on `identity(x)` not having a member `msg`. It falls and trips on the decorator. It is considered better practice to disable the warning locally than in the entire project by editing the `.pylintrc` file...